integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/CertificateDataConverter.java

```diff
@@ -102,6 +103,10 @@ private static String toLabel(CertificateDataElement element) {
 
   public CertificateQuestionValue toValue(CertificateDataElement element,
       List<CertificateDataElement> subQuestions) {
+    if (element.getConfig() instanceof CertificateDataConfigHeader) {
```

Comment:
To stay uniformed with the other if-statements in this method, would it be a good idea to create a describing method instead? Something like ```elementIsHeader()```

Also, maybe instead of returning null, we could return an CertificateQuestionValue without value? Or would that break something down the line?

```java
return CertificateQuestionValueText.builder().build()
```